client/my-sites/earn/memberships/index.tsx

```diff
@@ -44,12 +44,45 @@ import { ADD_NEWSLETTER_PAYMENT_PLAN_HASH, LAUNCHPAD_HASH } from './constants';
 
 import './style.scss';
 
-function MembershipsSection( { query } ) {
+type Query = {
+	stripe_connect_success?: string;
+	stripe_connect_cancelled?: string;
+};
+
+type MembershipsSectionProps = {
+	query: Query;
+};
+
+type Subscriber = {
+	id: string;
+	status: string;
+	start_date: string;
+	end_date: string;
+	// appears as both subscriber.user_email & subscriber.user.user_email in this file
+	user_email: string;
+	user: {
+		ID: string;
+		name: string;
+		user_email: string;
+	};
+	plan: {
+		connected_account_product_id: string;
+		title: string;
+		renewal_price: number;
+		currency: string;
+		renew_interval: string;
+	};
+	renew_interval: string;
+	all_time_total: number;
+};
```

Comment:
It seems like a Subscriber type should already exist somewhere, but I could not find one and created it here.

Reply:
@Automattic/apex any good generic place to add Subscriber's types?